.github/workflows/verify_library_generation.yaml

```diff
@@ -10,21 +10,35 @@ on:
 name: verify_library_generation_against_googleapis-gen
 jobs:
   verify_library_generation:
-    runs-on: ubuntu-22.04
     strategy:
       matrix:
         java: [ 8 ]
+        os: [ ubuntu-22.04, macos-12 ]
+    runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-java@v3
       with:
         java-version: ${{ matrix.java }}
         distribution: temurin
         cache: maven
+    - uses: kanga333/variable-mapper@3681b75f5c6c00162721168fb91ab74925eaebcb
```

Comment:
This is a personal project that has very few usage based on the Github stars, it looks risky to me and I'm not sure we want to use it. Is there any alternatives?

Reply:
> This is a personal project that has very few usage based on the Github stars, it looks risky to me and I'm not sure we want to use it.

Makes sense. I'll do more research then.

Reply:
@burkedavison thanks for the help.
However, Github complains that: Matrix options must only contain primitive values